clients/java/src/main/java/io/camunda/zeebe/client/impl/command/ActivateJobsCommandImpl.java

```diff
@@ -129,13 +130,13 @@ private void send(
 
   @Override
   public ActivateJobsCommandStep3 tenantId(final String tenantId) {
-    // todo(#13560): replace dummy implementation
+    builder.addTenantIds(tenantId);
```

Comment:
🔧 We should filter the list for duplicates to avoid unexpected behavior (also on line 139, and `StreamJobsCommandImpl#L138,144`)